app/src/main/java/ch/hikemate/app/model/facilities/FacilitiesRepository.kt

```diff
@@ -0,0 +1,115 @@
+package ch.hikemate.app.model.facilities
+
+import android.util.Log
+import ch.hikemate.app.model.route.Bounds
+import com.google.gson.Gson
+import com.google.gson.JsonObject
+import java.io.IOException
+import okhttp3.Call
+import okhttp3.Callback
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import org.osmdroid.util.GeoPoint
+
+class FacilitiesRepository(private val client: OkHttpClient) {
+
+  /**
+   * Sets a standard header for API requests.
+   *
+   * @param request Request builder instance to which headers will be added
+   */
+  private fun setRequestHeaders(request: Request.Builder) {
+    request.header("User-Agent", "Hikemate/1.0")
+  }
```

Comment:
Warning: There is the exact same function in the overpass API. This joins the global comment on this file.